packages/drivers/odsp-driver/src/opsCaching.ts

```diff
@@ -96,6 +99,7 @@ export class OpsCache {
 
             this.totalOpsToCache--;
             if (this.totalOpsToCache === 0) {
+                this.logger.sendPerformanceEvent({ eventName: "CacheOpsLimitHit"});
```

Comment:
@vladsud why are you putting this in the Performance table instead of the Generic table?

Reply:
It has impact on perf. Note - semantic here - what is the category of this event (there is no notion of tables here)
We can put it anywhere, but I find it more convenient for all perf-related events to be in one place